Userland/Libraries/LibC/getsubopt.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+// FIXME: This is a cleaned code (without C style conventions, for example) taken from glibc that
+// doesn't use C++ features at all. Try to adapt this code to use the language features to make
+// this code more understandable.
```

Comment:
Let's not import code from a third-party at this stage :)
Either adapt it enough so it's no longer recognisable as glibc, or completely write it from scratch. Additionally, you can't just put your own name & license into a file containing imported code.